libres/lib/include/ert/enkf/enkf_analysis.hpp

```diff
@@ -27,9 +27,18 @@
 
 #include <ert/enkf/obs_data.hpp>
 
-void enkf_analysis_fprintf_obs_summary(const obs_data_type *obs_data,
-                                       const meas_data_type *meas_data,
-                                       const char *ministep_name, FILE *stream);
+class UpdateSnapshot {
+public:
+    std::vector<std::string> obs_name;
+    std::vector<double> obs_value;
+    std::vector<double> obs_std;
```

Comment:
should be error, not std

Reply:
It is `std` everywhere else in the code, but it is of course not more correct with error